Don't define _XOPEN_SOURCE_EXTENDED and _XOPEN_SOURCE=600 at the same time

--- firefox-43.0b3/intl/icu/source/common/uposixdefs.h.~1~	2015-11-18 15:04:04.437741226 +0300
+++ firefox-43.0b3/intl/icu/source/common/uposixdefs.h	2015-11-18 15:04:22.383918698 +0300
@@ -52,9 +52,9 @@
  *
  * z/OS needs this definition for timeval and to get usleep.
  */
-#if !defined(_XOPEN_SOURCE_EXTENDED)
+/*#if !defined(_XOPEN_SOURCE_EXTENDED)
 #   define _XOPEN_SOURCE_EXTENDED 1
-#endif
+#endif */
 
 /*
  * There is an issue with turning on _XOPEN_SOURCE_EXTENDED on certain platforms.
